masterlist.yaml

```
@@ -23158,12 +23379,18 @@ plugins:
       - <<: *alreadyInX
         subs: [ 'imp_helm_legend_WACCF_AMB_Patch.esp' ]
         condition: 'active("imp_helm_legend_WACCF_AMB_Patch.esp")'
+      - <<: *alreadyInX
+        subs: [ 'Requiem - Improved Closefaced Helmets.esp' ]
+        condition: 'active("Requiem - Improved Closefaced Helmets.esp")'
       - <<: *useInstead
         subs: [ 'imp_helm_legend_WACCF_Patch.esp' ]
         condition: 'active("Weapons Armor Clothing & Clutter Fixes.esp")'
       - <<: *useInstead
         subs: [ 'imp_helm_legend_WACCF_AMB_Patch.esp' ]
         condition: 'active("Weapons Armor Clothing & Clutter Fixes.esp") and active("aMidianBorn_ContentAddon.esp")'
+      - <<: *useInstead
+        subs: [ 'Requiem - Improved Closefaced Helmets.esp' ]
```

Users may not know where this plugin is located, so it should be linked to:

```suggestion
        subs: [ '[Requiem - Improved Closefaced Helmets.esp](https://www.nexusmods.com/skyrimspecialedition/mods/64410/)' ]
```